dftimewolf/lib/logging_utils.py

```diff
@@ -102,4 +106,11 @@ def format(self, record: LogRecord) -> str:
       if loglevel_color:
         message = loglevel_color + message + RESET_SEQ
       record.msg = message
+
+    if self.threaded:
+      stack = [i[3] for i in inspect.stack()]
+      if 'Process' in stack:
+        record.msg = "[{0:s}] {1:s}".format(
+            threading.current_thread().getName(),
+            record.getMessage())
```

Comment:
Could you assign intermediary variables here? i.e. `thread_name` and `message`
 
I see you're using `record.getMessage()` again here, does this preserve the colorization that was made in the previous block?

Reply:
![image](https://user-images.githubusercontent.com/3506797/141017701-cceeb008-0f59-4f9c-8a79-b38ae1d73819.png)

Can confirm colour is preserved.